.github/CODEOWNERS

```
@@ -29,6 +29,7 @@
 /op-node        @ethereum-optimism/go-reviewers
 /op-node/rollup @protolambda @trianglesphere
 /op-proposer    @ethereum-optimism/go-reviewers
+/op-program     @ethereum-optimism/go-reviewers
```

```suggestion
/op-program     @ajsutton @protolambda
```
I think it makes more sense to assign just those who are assigned to the fault-proof program project. We can add more reviewers later if review is a bottleneck, and we'll soon have at least one more contributor to this project :).

---

If you assign people directly rather than using a team it will request reviews from everyone who is not the PR author. Unfortunately the UX isn't as good as a team.

I also lean to having the full team be able to review this. If they think they are missing context a reviewer (or PR author) can always unassign someone & assign someone else.

---

ok, I'm going to stick with the team and see how we go - I'll be watching all the changes anyway since it's a small codebase so I'm not concerned about quality.  And we can easily change if it proves a distraction for people or if you feel really strongly about it @protolambda